db/data_migrations/20181207103953_add_users_to_super_admin_team.rb

```diff
@@ -0,0 +1,23 @@
+class AddUsersToSuperAdminTeam < ActiveRecord::DataMigration
+  def up
+    team = Team.find_by_name('Super Admin')
+
+    return if team.nil?
+
+    users = []
+    users << User.find_by_email('andrien.ricketts@digital.justice.gov.uk')
+    users << User.find_by_email('alex.robinson@digital.justice.gov.uk')
+    users << User.find_by_email('sophie.osman@digital.justice.gov.uk')
+    users << User.find_by_email('anna.brennan@digital.justice.gov.uk')
+    users << User.find_by_email('daphne.haitas@digital.justice.gov.uk')
+    users << User.find_by_email('elliot.bouher@digital.justice.gov.uk')
+
+    users.each do |user|
+      next if user.nil?
+
+      next unless TeamMember.find_by_user_id(user.id).nil?
+
+      TeamMember.create(user_id: user.id, team_id: team.id, created_by_user_id: team.created_by_user_id)
+    end
```

Comment:
Is this right? you go to the next user if `TeamMember.find_by_user_id(user.id)` is not nil?  If I understand correctly, if the user is a member of another team, then you won't add them to the Super Admin team.

Reply:
I was actually meant to do this: **next unless TeamMember.find_by(user_id: user.id, team_id: team.id).nil?** To make sure that person was not already a member of the super_admin team. In tests I was able to add myself to the team more than once.